app/assets/javascripts/live_search.js

```diff
@@ -101,13 +101,18 @@
       pageUpdated.done(
         function(){
           var newPath = window.location.pathname + "?" + $.param(this.state);
+          this.trackingInit();
```

Comment:
it makes more sense to have it in this order

```
var newPath = window.location.pathname + "?" + $.param(this.state);
history.pushState(this.state, '', newPath);
this.trackingInit();
```

Reply:
👍